x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/discovery_file_dir_discovery.json

```diff
@@ -0,0 +1,43 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Enumeration of files and directories using built-in tools. Adversaries may use the information discovered to plan follow-on activity.",
+  "index": [
+    "logs-endpoint.events.*",
+    "winlogbeat-*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "File and Directory Discovery",
+  "query": "process where event.type in (\"start\", \"process_started\") and\n  (process.name : \"cmd.exe\" or process.pe.original_file_name == \"Cmd.Exe\") and\n  process.args : (\"dir\", \"tree\")\n\n",
```

Comment:
Is this an expected failure here?

![image](https://user-images.githubusercontent.com/2946766/101855571-e3b1ba80-3b20-11eb-986f-563f946a02b0.png)

Reply:
Looks like the EQL case-insensitive lists aren't working: `field : ("abc", "def")`.

This was with the latest version of ES from master, right?

Reply:
Yeah, had to fetch upstream into this branch to fully test the upgrade flow because https://github.com/elastic/kibana/pull/82969 got merged in-between the last build.

Reply:
So does that mean the latest ES isn't supporting this syntax?

Reply:
The test cluster es version looks to be a little behind on the snapshots (`2020-11-12`) -- do you know if support for this syntax was a recent addition?

``` ts
{
  "name" : "siem.estc.dev",
  "cluster_name" : "elasticsearch",
  "version" : {
    "number" : "8.0.0-SNAPSHOT",
    "build_flavor" : "default",
    "build_type" : "deb",
    "build_hash" : "7a890e8ea64d037c00606e42abdfeed05b252657",
    "build_date" : "2020-11-12T09:56:37.472505Z",
    "build_snapshot" : true,
    "lucene_version" : "8.7.0",
    "minimum_wire_compatibility_version" : "7.11.0",
    "minimum_index_compatibility_version" : "7.0.0"
  },
  "tagline" : "You Know, for Search"
}
```

Reply:
Looks like we're good here with the latest es snapshot @rw-access -- thank you! 🙂 

<p align="center">
  <img width="700" src="https://user-images.githubusercontent.com/2946766/102139394-042c8e00-3e1b-11eb-88c4-dc5fea9df78a.png" />
</p>


``` json
{
  "name" : "siem.estc.dev",
  "cluster_name" : "elasticsearch",
  "version" : {
    "number" : "8.0.0-SNAPSHOT",
    "build_flavor" : "default",
    "build_type" : "deb",
    "build_hash" : "3cbe0eadce4a7d12247bbbc0899350dca7cea3b6",
    "build_date" : "2020-12-10T10:01:49.580382Z",
    "build_snapshot" : true,
    "lucene_version" : "8.7.0",
    "minimum_wire_compatibility_version" : "7.11.0",
    "minimum_index_compatibility_version" : "7.0.0"
  },
  "tagline" : "You Know, for Search"
}
```